### PR TITLE
Update project dictionary

### PR DESCRIPTION
**Pull request by catgirlseraid**
## What
Update the project dictionary to reduce the amount of typos shown

exclude_from_changelog

### CODE REVIEW

**Comment by Thunderblade73** [comment]
| Without This | With This |
| ---- | ----|
|  image | image |

**Review by Thunderblade73 — CHANGES_REQUESTED** [review]
You missed to add to the git ignore
image
(Screenshot from hannibal)

**Comment by CalMWolfs** [comment]
> You missed to add to the git ignore

done

**Review by Thunderblade73 — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .idea/*
 !.idea/icon.svg
+!.idea/dictionaries
```
Comment (Thunderblade73) [inline .gitignore]: please no, make the exact file, or else we will collect a lot of files over time

**Review by Thunderblade73 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (CalMWolfs) [reply .gitignore]: done x2

**Review by Thunderblade73 — APPROVED** [review]

**Review by CalMWolfs — APPROVED** [review]

**Comment by hannibal002** [comment]
<table>
<tr>
 <td>Current
 <td>Added bunch of words
<tr>
 <td>

image

 <td>

image

</table>

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**